src/telegram/process_update_task.rs

```diff
@@ -1,11 +1,17 @@
+use crate::db::{ClientState, DbController};
+use crate::{BotError, BINARY_FILE, OPEN_WEATHER_MAP_API_TOKEN, RUST_TELEGRAM_BOT_TOKEN};
 use fang::async_trait;
 use fang::asynk::async_queue::AsyncQueueable;
 use fang::asynk::AsyncError as Error;
-use fang::serde::Deserialize;
-use fang::serde::Serialize;
+use fang::serde::{Deserialize, Serialize};
```

Comment:
can we split imports? it's easier to read and search for them

Reply:
Sure :D

Reply:
Done